integration-cli/docker_cli_kill_test.go

```diff
@@ -49,11 +44,6 @@ func (s *DockerSuite) TestKillDifferentUserContainer(c *check.C) {
 
 	cleanedContainerID := strings.TrimSpace(out)
 
-	inspectCmd := exec.Command(dockerBinary, "inspect", cleanedContainerID)
```

Comment:
LOL the time you commented I force pushed with the exact modification you had in mind